app/redux/cart.js

```diff
@@ -116,14 +116,29 @@ export const fetchCart = () => {
 export const newSessionCart = () => {
   return (dispatch, getState, { axios }) => {
     return axios.post('/api/cart/')
-      .then(response => {
-        return response.data
-      })
+      .then(response => response.data)
       .then(cart => dispatch(setCart(cart)))
       .catch(e => console.log(chalk.red(`Error in Redux thunk createSessionCart: ${e}`)))
   }
 };
 
+export const mergeCart = () => {
+  return (dispatch, getState, { axios }) => {
+    // grab id of user cart
+    return axios.put('/api/cart')
```

Comment:
Is the purpose of the PUT call here to get the ID of the current user's cart?
GET(/api/cart) already does that. Looks like there's some duplicated logic here.

Reply:
Yep, just fixed it with new push